db/migrate/20190115215627_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[5.2]
+  def change
+    create_table :comments do |t|
+      t.string :commenter
```

Comment:
We don't use the commenter data column anywhere, and we don't even unit test it. Please remove it